### ISSUE TITLE
Improvement for the iHealth BP5 Blood pressure monitor Meteor plugin

### ISSUE THREAD

**Issue by ihealthdavid** [iHealthLab/test-labs#2]
API request

1. (new) BpManagerCordova.connectedDevice("8CDE52143F1E", success, failure, "");
- return details on the connected device.
- if not connected, fail and return meaningful message

2. (new) Is there a way the API could alert the app if a connected device is disconnected or encounters an error?

3. (change) BpManagerCordova.search("", success, failure, "")
- return all devices, including connected device
- consider returning some more info, but we will review after checking #4

4. (new) BpManagerCordova.searchDebug ("", success, failure, "")
- return all devices, including connected device
- consider returning all details available to understand what info is available. or if you have doc and possible information that would help.

5. (new)
a) BpManagerCordova.searchOnly ("", success, failure, "")
b)  BpManagerCordova.connectBlueToothOnly ("8CDE52143F1E", success, failure, "")
c)  BpManagerCordova.connectiOSstreamOnly ("8CDE52143F1E", success, failure, "")
c)  BpManagerCordova.authenticateOnly ("8CDE52143F1E", success, failure, "")
- separate functionality for #3 BpManagerCordova.search

**Comment by kevingzhang** [comment]
You need to send this to XueWei. I did not see him in this issue participants.

**Comment by ihealthdavid** [comment]
Yes, I spoke to him in person. We are currently in a meeting with yuchen.
Not sure how to implement disconnect.

**Comment by ihealthdavid** [comment]
@kakahero FYI
